ui/components/app/user-preferenced-token-input/user-preferenced-token-input.component.js

```diff
@@ -18,7 +18,7 @@ export default class UserPreferencedTokenInput extends PureComponent {
     return (
       <TokenInput
         {...restProps}
-        showFiat={!useNativeCurrencyAsPrimaryCurrency}
+        showFiat={useNativeCurrencyAsPrimaryCurrency}
```

Comment:
Is it as easy as flipping this?  Can we write a test?

Reply:
@darkwing done , i just added the test